tests/e2e/Steps/actionSteps.js

```diff
@@ -0,0 +1,13 @@
+import {config} from "../wdio.conf";
+export class Steps {
```

Comment:
You can export not class, but object. In this case `new Steps()` won't be required in external files.